npm/angular/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "dist/index.js",
   "scripts": {
     "prebuild": "rimraf dist",
-    "build": "tsc || echo 'built, with type errors'",
+    "build": "rollup -c rollup.config.js",
```

Comment:
Paired with Jordan on this, we fixed the Angular cleanup and needed to add `@cypress/mount-utils` for the cleanup. We needed to revert back to rollup so that the `@cypress/mount-utils` was internalized.